relay-server/src/services/outcome.rs

```diff
@@ -251,6 +252,79 @@ impl fmt::Display for Outcome {
     }
 }
 
+/// A lower-cardinality version of [`RuleId`].
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub enum RuleCategory {
+    BoostLowVolumeProjects,
+    BoostEnvironments,
+    IgnoreHealthChecks,
+    BoostKeyTransactions,
+    Recalibration,
+    BoostReplayId,
+    BoostLowVolumeTransactions,
+    BoostLatestReleases,
+    Custom,
+    Other,
+}
+
+impl RuleCategory {
+    fn to_str(self) -> &'static str {
+        match self {
+            Self::BoostLowVolumeProjects => "1000",
+            Self::BoostEnvironments => "1001",
+            Self::IgnoreHealthChecks => "1002",
+            Self::BoostKeyTransactions => "1003",
+            Self::Recalibration => "1004",
+            Self::BoostReplayId => "1005",
+            Self::BoostLowVolumeTransactions => "1400",
+            Self::BoostLatestReleases => "1500",
+            Self::Custom => "3000",
+            Self::Other => "?",
+        }
+    }
+}
+
+impl From<RuleId> for RuleCategory {
+    fn from(value: RuleId) -> Self {
+        match value.0 {
+            1000 => Self::BoostLowVolumeProjects,
+            1001 => Self::BoostEnvironments,
+            1002 => Self::IgnoreHealthChecks,
+            1003 => Self::BoostKeyTransactions,
+            1004 => Self::Recalibration,
+            1005 => Self::BoostReplayId,
+            n if (1400..1500).contains(&n) => Self::BoostLowVolumeTransactions,
```

Comment:
```suggestion
            1400..=1499 => Self::BoostLowVolumeTransactions,
```

And for the following lines the same.